docs/contributing/README.md

```diff
@@ -8,16 +8,16 @@ Want to contribute to the project? That's great to hear! You can find more infor
 
 ## Why to contribute?
 
-As you may know, Frontity is an open source project, available and free for everyone. We believe that working in an open, collaborative way and sharing knowledge has the power to change the world for the better.
+As you may know, Frontity is an open source project, available and free for everyone. We believe that working in an open way and sharing knowledge has the power to change the world for the better.
 
-Here are some good reasons to contribute to open source:
+Here are some more good reasons to contribute to open source:
 
-* **Connects you with people:** an open source community usually consists of people sharing similar interests, so it will allow to connect with amazing people.
-* **Improve your skills and competences:** if you are looking to practice, there is always a task for you in open source projects. This will allow you improve your knowledge.
-* **Learn from mentors and share your thoughts:** it is an incredible way to share and get expertise, and it is really refreshing.
+* **Build new connections**: an open source community usually consists of people sharing similar interests, so contributing to a project will allow you to connect with amazing people.
+* **Improve your skills and competences**: if you are looking for practice, there is always a task for you in open source projects. This will allow you improve your knowledge.
```

Comment:
@rmartinezduque "if you are looking to practice" was correct, though I would rephrase as "if you want to practice your skills..."